yarn.lock

```diff
@@ -2444,6 +2444,25 @@ __metadata:
   languageName: node
```

Comment:
could you revert this `yarn.lock` change?

Reply:
@P0lip I reverted, but it was very strange, because without execution of `yarn` tests were failing. Let's see if it passes this time.